packages/notification-center-vue/src/lib/notification-center-component/NotificationCenterComponent.vue

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
```

Comment:
this is a Vue wrapper around the NC Web Component...
I've actually also removed Stencil generated files as we don't need them at all